tests/unit/components/Results/SearchLinkChips.spec.js

```diff
@@ -0,0 +1,119 @@
+import {createLocalVue, shallowMount} from "@vue/test-utils";
+import Vuetify from "vuetify"
+import SearchLinkChips from "@/components/Results/SearchLinkChips.vue"
+
+let $route = {
+  name: "search",
+  query: {}
+};
+
+const $router = {
+  push: jest.fn(),
+};
+
+describe("SearchLinkChips.vue", function () {
+  let wrapper;
+  const vuetify = new Vuetify();
+  const localVue = createLocalVue();
+
+  beforeEach(() => {
+    wrapper = shallowMount(SearchLinkChips, {
+      localVue,
+      vuetify,
+      propsData: {
+        record: {
+          name: 'test record',
+          subjects: [
+            {
+              label: 'first chip',
+              active: false
+            }
+          ],
+          domains: [
+            {
+              label: 'second chip',
+              active: false
+            }
+          ]
+        }
+        /*
+        type: 'domains',
+        chips: [
+          {
+            label: 'first chip',
+            active: false
+          }
+        ]
+         */
+      },
+      mocks: {$route, $router}
+    });
+  });
+
+  it("can be instantiated", () => {
+    expect(wrapper.name()).toMatch("SearchLinkChips");
```

Comment:
Please use ```wrapper.vm.$options.name``` instead of ```wrapper.name()``` as later one is deprecated and it is giving console error while running test